source/accessibility-statement.html.erb

```diff
@@ -4,96 +4,144 @@ title: Accessibility statement for GOV.UK&nbsp;Pay
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="/">GOV.UK&nbsp;Pay</a>
-        </li>
-        <li class="govuk-breadcrumbs__list-item">
-            <a class="govuk-breadcrumbs__link" href="#main-content" aria-current="page"><%= current_page.data.title %></a>
-        </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/service-toolkit#gov-uk-services">GOV.UK services</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="/">GOV.UK&nbsp;Pay</a>
+      </li>
+      <li class="govuk-breadcrumbs__list-item">
+        <a class="govuk-breadcrumbs__link" href="#main-content" aria-current="page"><%= current_page.data.title %></a>
+      </li>
     </ol>
   </div>
   <main id="main-content" class="govuk-main-wrapper" role="main">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l" id="heading-1">Accessibility statement for GOV.UK&nbsp;Pay</h1>
-      <p class="govuk-body">This accessibility statement explains how accessible this service is, what to do if you have difficulty using it, and how to report accessibility problems with the service.</p>
+        <h1 class="govuk-heading-l" id="heading-1">Accessibility statement for GOV.UK&nbsp;Pay</h1>
+        <p class="govuk-body">This accessibility statement explains how accessible this service is, what to do if you
+          have difficulty using it, and how to report accessibility problems with the service.</p>
 
-      <p class="govuk-body">This service is part of the wider GOV.UK website. There’s a separate <a class="govuk-link" href="https://www.gov.uk/help/accessibility">accessibility statement for the main GOV.UK website</a>. There is also a separate <a class="govuk-link" href="https://docs.payments.service.gov.uk/accessibility.html">accessibility statement for GOV.UK&nbsp;Pay's technical documentation</a>.</p>
+        <p class="govuk-body">This service is part of the wider GOV.UK website. There’s a separate
+          <a class="govuk-link" href="https://www.gov.uk/help/accessibility">accessibility statement for the main GOV.UK
+            website</a>. There is also a separate
+          <a class="govuk-link" href="https://docs.payments.service.gov.uk/accessibility.html">accessibility statement
+            for GOV.UK&nbsp;Pay's technical documentation</a>.</p>
 
-      <p class="govuk-body">This page only contains information about the GOV.UK&nbsp;Pay service, available at <a class="govuk-link" href="https://www.payments.service.gov.uk">https://www.payments.service.gov.uk</a>.</p>
+        <p class="govuk-body">This page only contains information about the GOV.UK&nbsp;Pay service, available at
+          <a class="govuk-link" href="https://www.payments.service.gov.uk">https://www.payments.service.gov.uk</a>.</p>
```

Comment:
We seem to be using a mixture of fully qualified URLs and relative paths in our links - should we pick one or the other?

Reply:
we only use relative links for any link between pages, this is outdated but is also the only example where we don't because it is literally readable as a fully qualified link on the page